assets/js/googlesitekit/api/index.js

```diff
@@ -45,16 +124,22 @@ export const get = async (
  *
  * @return {Promise} A promise for the `fetch` request.
  */
-// eslint-disable-next-line no-unused-vars
 export const set = async (
 	type,
 	identifier,
 	datapoint,
 	data,
-	// eslint-disable-next-line no-unused-vars
-	{ disableCache = false, queryParams = {} } = {}
+	{ method = 'POST', queryParams = {} } = {}
 ) => {
-	throw new Error( 'Not yet implemented.' );
+	return siteKitRequest( {
```

Comment:
There's one piece missing here in this function: Based on our routes, a successful `set` call should afterwards call `invalidateCache()` for the same `type`, `identifier`, and `datapoint` it was called for, because setting it typically affects the value that getting it would result in. This is the current behavior and should be maintained.

Reply:
Oh, right! Thanks, I'll get that sorted 😄

Reply:
Added! 😄